spectacles/cli.py

```diff
@@ -157,10 +158,11 @@ def wrapper(*args, **kwargs):
 def set_file_handler(directory: str) -> None:
 
     global LOG_FILEPATH
+    global LOG_DIRECTORY
```

Comment:
Don't think we need both of these to be global. Can remove `LOG_FILEPATH` as global and just use `LOG_DIRECTORY / LOG_FILENAME` as a replacement.